config/application.rb

```diff
@@ -216,8 +220,22 @@ class Application < Rails::Application
     config.session_store :active_record_store, :secure => !!SETTINGS[:require_ssl]
   end
 
-  def self.setup_console
-    ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
-    puts "For some operations a user must be set, try User.current = User.first"
+  class << self
+    def setup_console
+      ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
+      puts "For some operations a user must be set, try User.current = User.first"
+    end
+
+    def dynflow
+      return @dynflow if @dynflow.present?
```

Comment:
Any time you call `Foreman.dynflow` after the first time, it will be set (and it will be equal to the world defined in the lines below). 

E.g: RssNotifications calls `perform_later` (which calls Foreman.dynflow.world inditrectly). The next time a job runs, the variable will be fetched from here.